.github/workflows/build_deploy_exm_fe.yml

```diff
@@ -0,0 +1,89 @@
+name: Build and Deploy EXM FE
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build:
+    name: Build Angular Application
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Download and install Maven
+      run: sudo apt-get update && sudo apt-get install -y maven
+
+    - name: Build Angular application using Maven
+      run: |
+        export BE_BASE_URL=http://backend-url-here
```

Comment:
Vreau sa adaugi asta la comanda de mvn clean install, cauta in documentatie la maven la cum poti da variabila la comanda, nu o sa te ajute exportul aici

Reply:
Si o sa vrem la backend-url-here sa punem ip-ul de la backend si port-ul cred, deci http://ip:8080